### PR TITLE
remove google plus

### PR DESCRIPTION
**Pull request by MarcoFalke**
Google plus was shut down long ago, so no need to keep any references to it. Also, remove other google settings.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -30,13 +30,6 @@ owner:
...
-  google:
-    plus: #username
-    analytics: #UA-85008990-1
-    verify:
-    ad-client:
-    ad-slot:
-  bing-verify:
   github: bitcoin
   stackoverflow: #123456/username   from a "http://stackoverflow.com/users/123456/username" link
   linkedin: #username
```
Comment (laanwj) [inline _config.yml:35]: I guess if we're deleting unnecessary settings, there's quite a few below here as well which could go (all of them? I don't see a bitcoin core steam release happening any time soon…).

File: _config.yml [hunk 2]
```diff
@@ -25,34 +25,7 @@ kramdown:
 # Site owner
 owner:
   name: Bitcoin Core
-  avatar: bio-photo.jpg
```
Comment (laanwj) [inline _config.yml:28]: Asset unused after this?

File: _data/authors.yml [hunk 3]
```diff
@@ -2,40 +2,19 @@
```
Comment (laanwj) [inline _data/authors.yml:2]: Is `authors.yml` still used at all?

Comment (MarcoFalke) [reply _data/authors.yml:2]: Yes, for author.github, author.twitter and author.name. Example: https://bitcoincore.org/en/2015/12/14/segregated-witness/

I'll remove that in a follow-up.

File: _config.yml (hunk 2 above)
Comment (MarcoFalke) [reply _config.yml:28]: Not yet. Still used in _includes/author-bio.html

I'll remove that in a follow-up.

File: _data/authors.yml (hunk 3 above)
Comment (laanwj) [reply _data/authors.yml:2]: Thanks!

**Comment by MarcoFalke** [comment]
ci is green

**Comment by katesalazar** [comment]
concept ACK, looking forward to review eventually

**Comment by Zero-1729** [comment]
Concept ACK

**Comment by katesalazar** [comment]
approach ACK, untested

**Comment by MarcoFalke** [comment]
rfm? :)

**Pull request merged**